platform/golden/bullseye.pkr.hcl

```diff
@@ -1,59 +1,107 @@
-variables {
-  headless          = true
-  iso_checksum      = "224cd98011b9184e49f858a46096c6ff4894adff8945ce89b194541afdfd93b73b4666b0705234bd4dff42c0a914fdb6037dd0982efb5813e8a553d8e92e6f51"
-  iso_checksum_type = "sha512"
-  iso_url           = "https://cdimage.debian.org/debian-cd/11.6.0/amd64/iso-cd/debian-11.6.0-amd64-netinst.iso"
+variable "config_file" {
+  type    = string
+  default = "debian-11-preseed.cfg"
+}
+
+variable "config_dir" {
+  type    = string
+  default = "cfg"
```

Comment:
I have a PR which sets this properly. Would you consider:
- merge https://github.com/artificialwisdomai/origin/pull/8
- merge other outstanding PRs
- rebase your PR on those PRs
- set the path properly as per above pr.

Reply:
Yep

Reply:
updated to use the root path in the http_directory builder call
Kept the target directory variable.